inc/CLI/LanguagePackCommand.php

```diff
@@ -233,19 +225,31 @@ protected function check_optional_args_and_all( $args, $all ): array {
 			WP_CLI::success( 'No projects found' );
 		}
 
-		$args = array_map(
-			function ( $project ) {
-				$project = ( new ProjectLocator( $project ) )->get_project();
-				if ( $project->is_active() ) {
-					return $project;
+		$projects = [];
+		if ( $all ) {
+			$projects = $this->get_all_projects();
+		}
+
+		$projects = array_map(
+			function ( $project_id ) {
+				$project = ( new ProjectLocator( $project_id ) )->get_project();
+				if ( ! $project ) {
+					WP_CLI::log( sprintf( 'Project (ID: %d) does not exist.', $project_id ) );
+					return null;
 				}
-				WP_CLI::log( sprintf( 'Project (ID: %d) is inactive.', $project->get_id() ) );
-				return null;
+				if ( ! $project->is_active() ) {
+					WP_CLI::log( sprintf( 'Project (ID: %d) is inactive.', $project->get_id() ) );
+					return null;
+				}
+				return $project;
 			},
 			$args
 		);
 
-		return $args;
+		// Remove inexistant projects.
```

Comment:
```suggestion
		// Remove inexistent projects.
```

Reply:
Rare :(

![kép](https://user-images.githubusercontent.com/952007/125000671-31f45100-e051-11eb-9cb3-fedee70a779c.png)

```suggestion
		// Remove non-existent projects.
```